empire/pkg/runner/runner.go

```diff
@@ -0,0 +1,178 @@
+// package runner provides a simple interface for running docker containers.
```

Comment:
Already talked to you in personal about this so just putting it here to document.

`package runner` just provides an easy interface for running "attached" containers; where you want to run some command in an image with attached stdin, stdout, stderr.

This package just gets consumed as a primitive by `package service` for running attached processes. It would be nice if we can eventually implement this in such a way that even attached containers get run by ECS...